addons/test_mass_mailing/models/mailing_models.py

```diff
@@ -4,6 +4,36 @@
 from odoo import api, fields, models
 
 
+class MailingSimple(models.Model):
```

Comment:
Is it ok to have the same class name between this one and the real "mail.test.simple" ?

(Note that description is also copy/pasta'ed)

Reply:
Updated class name. But docstring is not copy pasted, it is different.

Reply:
(I was taking about "_description", not docstring. Not very important but heh)